astro_tests/dags/.airflowignore

```diff
@@ -0,0 +1 @@
+dags/
```

Comment:
Wait how does this work? You're putting the DAGs directory in the .airflowignore? How does airflow find the DAGs then?

Reply:
Weird, I don't remember adding this to the `.airflowignore`, I'll remove it

Reply:
I just realized I must've fat fingered this. I meant for it to have `dbt/` in there. @dimberman wouldn't that make more sense?